pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -455,3 +458,49 @@ func aggregatePersistentVolumeClaimStatus(object *unstructured.Unstructured, agg
 	pvc.Status = *newStatus
 	return helper.ToUnstructured(pvc)
 }
+
+func aggregatePodDisruptionBudgetStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	pdb := &policyv1.PodDisruptionBudget{}
+	err := helper.ConvertToTypedObject(object, pdb)
+	if err != nil {
+		return nil, err
+	}
+
+	newStatus := &policyv1.PodDisruptionBudgetStatus{
+		DisruptedPods: make(map[string]metav1.Time),
+	}
+	for _, item := range aggregatedStatusItems {
+		if item.Status == nil {
+			continue
+		}
+
+		temp := &policyv1.PodDisruptionBudgetStatus{}
+		if err = json.Unmarshal(item.Status.Raw, temp); err != nil {
+			return nil, err
+		}
+		klog.V(3).Infof(
+			"Grab pdb(%s/%s) status from cluster(%s), desired healthy: %d, current healthy: %d, disrupted: %d, expected: %s",
+			pdb.Namespace, pdb.Name, item.ClusterName,
+			temp.DesiredHealthy, temp.CurrentHealthy, temp.DisruptedPods, temp.ExpectedPods,
+		)
+
+		newStatus.CurrentHealthy += temp.CurrentHealthy
+		newStatus.DesiredHealthy += temp.DesiredHealthy
+		newStatus.ExpectedPods += temp.ExpectedPods
+		newStatus.DisruptionsAllowed += temp.DisruptionsAllowed
+		newStatus.Conditions = append(newStatus.Conditions, temp.Conditions...)
+		if temp.DisruptedPods != nil {
+			for k, v := range temp.DisruptedPods {
```

Comment:
Is it possible that different clusters have the same pod name?

Reply:
UID is guaranteed to be unqiue across a whole cluster. To have a collision on a replicaset UID and a pod UID is highly highly unlikely.

pod name: `{deployment-name}-{replicaset-uid}-{pod-uid}`

I'll go ahead and include the clusterName in there as well

Reply:
ok, thanks~